### ISSUE TITLE
Proposed maintenance cycle issue template

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#414]
The goal of this template is to outline the steps needed when performing maintenance on the frontend. Maintenance should be performed on a regular schedule to ensure the frontend is up to date and secure.

**Tasks:**
A list of tasks to be completed for the maintenance cycle. The following steps should take place each cycle.

- [ ] run `yarn audit` to determine what packages have security issues and patches
- [ ] Determine which packages to update and list them
- [ ] List the ignored updates in the **Ignore** section of this ticket
- [ ] Organize maintenance tickets to be completed
- [ ] A checklist of tasks to complete. This list will be updated and approved before work begins on the maintenance.

**Ignore:**

- A list of tasks to be ignored. For example, a low priority security patch that is not necessary for updating.

**Dependencies/Blocks:**

- Blocked By: Issues that are blocking the completion of the current issue.
- Blocking: Issues being blocked by the completion of the current issue.

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**

- These links can consist of resources, bugherds, etc.

**Needed for merging:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Execute unit tests
- [ ] Execute integration tests
- [ ] Update documentation, if needed
- [ ] Test the build

**Notes:**
Notes from the developer on this maintenance cycle.

**Comment by roamye** [comment]
@kamerynB - Is there anything missing from this template? or can it go into prioritization review to receive feedback from core team?

**Comment by zachjschwartz** [comment]
@kamerynB I know this is an old one, but should this go into future work if it's just going to sit?

**Comment by jffcamp** [comment]
@kamerynB , I've added to this opening block to say priority should be given to critical and high vulnerabilities. Looks good.

**Comment by kamerynB** [comment]
@jffcamp @prowns @zachjschwartz I am going to close this issue as it was used as a template for building the maintenance GitHub issue template. It is no longer needed.
